subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -1421,8 +1496,8 @@ static void ticker_op_update_cb(uint32_t status, void *param)
 }
 
 #if defined(CONFIG_BT_PERIPHERAL)
-static void ticker_stop_cb(uint32_t ticks_at_expire, uint32_t remainder, uint16_t lazy,
-			   void *param)
+static void ticker_stop_cb(uint32_t ticks_at_expire, uint32_t remainder,
+	uint16_t lazy, void *param)
```

Comment:
Use leading tabs, and then spaces to align with parameters.

Reply:
(See previous comment.)